react-tutorial/src/Board.js

```diff
@@ -0,0 +1,76 @@
+import { useState } from "react";
```

Comment:
In order to have a better structure I recommend you to move Board and Square to a "components" folder